pkgs/applications/misc/epr/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, python3Packages, fetchFromGitHub, ncurses }:
```

Comment:
ncurses doesn't seem to be used in the package description?

Reply:
It looks like python is already finding the curses package

Reply:
Okay, thanks for pointing out!